### DISCUSSION TITLE
Question regarding EFIstub

### DISCUSSION THREAD

**Discussion by brkeejp** [sebanc/brunch#699] [General]
Don't know where to write soo I'm mentioning here. (and giving others tutorial how can they bypass grub)
I'm using EFIstub instead grub loader (for bypassing grup screen of course) but safe to use EFIstub on brunch right? (I don't see any impact for now but just asking)

PS: Thanks for reading (and for brunch of course) I'm your regular everyday user soo don't mind my question if you found it stupid. 

Here what change I made:

```sh
# X = Installed device
# Y = EFI Partition number (ESP)
# Z = Root Partition number

# L = Label what you want (mine is 'Chrome OS')
# U = ESP Partition UUID

sudo mkdir /mnt/esp /mnt/root

sudo mount /dev/sdXY /mnt/esp
sudo mount /dev/sdXZ /mnt/root

sudo mkdir /mnt/esp/EFI/chromeos

sudo cp /mnt/root/kernel /mnt/esp/EFI/chromeos/
sudo cp /mnt/root/initramfs.img /mnt/esp/EFI/chromeos/
sudo cp /mnt/root/lib/firmware/amd-ucode.img /mnt/esp/EFI/chromeos/
sudo cp /mnt/root/lib/firmware/intel-ucode.img /mnt/esp/EFI/chromeos/

sudo efibootmgr --disk /dev/sdX --part Y --create --label 'L' --loader /EFI/chromeos/kernel --unicode 'root=PARTUUID=UUUUUUUU-UUUU-UUUU-UUUU-UUUUUUUUUUUU rw boot=local noresume noswap loglevel=7 disablevmx=off cros_secure cros_debug console= cros_debug enforce_hyperthreading=1 options=no_ui_delay,enable_sensors,acpi_power_button vt.global_cursor=0 brunch_bootsplash=default initrd=\EFI\chromeos\amd-ucode.img initrd=\EFI\chromeos\intel-ucode.img initrd=\EFI\chromeos\initramfs.img' --verbose
```

**Comment by sebanc** [comment]
This works, however you will have to re-perform it each time you update brunch.

**Reply by brkeejp** [reply]
if working there is no problem. I can redo  steps.
Edit: thanks BTW

**Reply by IlhamAF2002** [reply]
Hello, I try to run the code after entering TTY mode but failed. It is said '/mnt/root/kernel' no such file or dictionary. I have mounted the root partition on /mnt/root before( As I do not dual boot,  I mount nvme0n1p3) but still failed. Can you help me ?

**Comment by IlhamAF2002** [comment]
Hello, I try to run the code after entering TTY mode but failed. It is said '/mnt/root/kernel' no such file or dictionary. I have mounted the root partition on /mnt/root before( As I do not dual boot, I mount nvme0n1p3) but still failed. May you help me ?
